torch/distributed/device_mesh.py

```diff
@@ -349,6 +349,81 @@ def _setup_world_group_and_device(self):
 
             return _get_default_group()
 
+        @staticmethod
+        def _init_one_process_group(
+            sub_layout: _MeshLayout,
+            rank_map: torch.Tensor,
+            dim_name: str,
+            backend_override: BackendConfig,
+        ) -> Optional[str]:
+            # swap the current dim to the last dim
+            # then reshape to flatten out other dims
```

Comment:
Nit: I think this comment is outdated, and it was probably me who forgot to remove it when I changed this code.